be/src/pipeline/pipeline_x/pipeline_x_fragment_context.h

```diff
@@ -156,6 +156,7 @@ class PipelineXFragmentContext : public PipelineFragmentContext {
 
     std::map<UniqueId, RuntimeState*> _instance_id_to_runtime_state;
     std::mutex _state_map_lock;
+    std::map<int, std::vector<PipelinePtr>> _union_child_pipelines;
```

Comment:
`_union_child_pipelines` has the similar function to `_build_side_pipelines`. Can you unify those two?